From 914422a3a0f73f4f26aed5f3e4087585cfb27df2 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Wed, 7 Nov 2018 19:25:51 +0800
Subject: [PATCH 5048/5242] MLK-20249 ARM64: dts: fsl-imx8mq-evk.dts: change
 the i2c bus pad to slow slew rate

commit  e86788880c403e01ec1979b39e8504c315bdeca4 from
https://source.codeaurora.org/external/imx/linux-imx.git

i2c2 and i2c3 are work at 100KHz, so no need to set the pad to fast
slew rate, slow slew rate is enough.

This patch can also fix the synaptics_dsx touch work unstable issue.
When config the i2c bus pad to fast slew rate, synaptics_dsx touch
sometimes can't be recognized through i2c bus. Seems this touch
i2c slave device sensitive to the pad slew rate setting.

Acked-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 683cd99..4c0c061 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -199,15 +199,15 @@
 
 		pinctrl_i2c2: i2c2grp {
 			fsl,pins = <
-				MX8MQ_IOMUXC_I2C2_SCL_I2C2_SCL			0x4000007f
-				MX8MQ_IOMUXC_I2C2_SDA_I2C2_SDA			0x4000007f
+				MX8MQ_IOMUXC_I2C2_SCL_I2C2_SCL			0x40000067
+				MX8MQ_IOMUXC_I2C2_SDA_I2C2_SDA			0x40000067
 			>;
 		};
 
 		pinctrl_i2c3: i2c3grp {
 			fsl,pins = <
-				MX8MQ_IOMUXC_I2C3_SCL_I2C3_SCL			0x4000007f
-				MX8MQ_IOMUXC_I2C3_SDA_I2C3_SDA			0x4000007f
+				MX8MQ_IOMUXC_I2C3_SCL_I2C3_SCL			0x40000067
+				MX8MQ_IOMUXC_I2C3_SDA_I2C3_SDA			0x40000067
 			>;
 		};
 
-- 
1.7.9.5

